examples/typescript/informer/informer.ts

```diff
@@ -0,0 +1,26 @@
+import * as k8s from '@kubernetes/client-node';
+
+const kc = new k8s.KubeConfig();
+kc.loadFromDefault();
+
+const k8sApi = kc.makeApiClient(k8s.CoreV1Api);
+
+const listFn = (fn) => {
+    k8sApi.listNamespacedPod('default')
+        .then((res) => {
+            const podList = res.body;
+            fn(podList.items, podList.metadata!.resourceVersion);
+        })
+        .catch((err) => {
+            // tslint:disable-next-line:no-console
```

Comment:
We should really just disable console statements for the entire file for our examples.

Reply:
done.